ios/FluentUI/Other Cells/ActionsCell.swift

```diff
@@ -95,18 +95,18 @@ open class ActionsCell: UITableViewCell {
     private var action1Type: ActionType = .regular
     private var action2Type: ActionType = .regular
 
-    private let topSeparator = Separator(style: .default, orientation: .horizontal)
-    private let bottomSeparator = Separator(style: .default, orientation: .horizontal)
-    private let verticalSeparator = Separator(style: .default, orientation: .vertical)
+    private let topSeparator = MSFDivider()
+    private let bottomSeparator = MSFDivider()
+    private let verticalSeparator = MSFDivider(orientation: .vertical)
 
     public override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
         super.init(style: style, reuseIdentifier: reuseIdentifier)
 
         contentView.addSubview(action1Button)
         contentView.addSubview(action2Button)
-        contentView.addSubview(verticalSeparator)
-        addSubview(topSeparator)
-        addSubview(bottomSeparator)
+        contentView.addSubview(verticalSeparator.view)
```

Comment:
so for this change, we should do a performance analysis with 100s of tableview cells scrolling and loading and see the impact of swiftuiview inside tableviewcell.